contents/handbook/strategy/strategy.md

```diff
@@ -70,7 +73,7 @@ What we will build:
 
 Making money:
 
-The first 6-12 months we will focus solely on developer adoption. After that, we'll slowly and carefully start introducing paid features for our enterprise customers. Those will include
+The first 6-12 months we will focus solely on developer adoption. After that, we'll slowly and carefully start introducing paid features for our enterprise customers. Those will include:
```

Comment:
... on developer adoption, whilst setting up the repo structure and a handful of enterprise features, to get us into the habit of managing expectations carefully with the community, and so we don't feel like we are focussed on revenue OR growth. The former will come from the latter  organically.